src/store/cloudApi.ts

```diff
@@ -0,0 +1,33 @@
+import { emptyCloudApi as api } from './emptyCloadApi';
+
+const sampleBlueprints = {
+  data: [
+    {
+      id: '1',
+      name: 'Blueprint 1',
+      version: '1.0',
+      description: '70% Dark Chocolate Blueprint 1',
+      last_modified_at: '2021-09-09T14:38:00.000Z',
+    },
+    {
+      id: '2',
+      name: 'Blueprint 2',
+      version: '1.1',
+      description: '70% Dark Chocolate Blueprint 2',
+      last_modified_at: '2021-09-09T14:38:00.000Z',
+    },
+  ],
+};
+
+// initialize an empty api service that we'll inject endpoints into later as needed
+export const cloudApi = api.injectEndpoints({
```

Comment:
We should probably call this the `blueprintsReducer`. We'll add the `cloudApi` later but we will need to handle all the blueprints related queries ourselves using Cockpit and files.